string name

bool always_on

string plugin_name
string plugin_file

string collision